File: front/src/app/shared/models/Empleado.ts
export class Empleado {
  public id: number;           // Id_Empleado
  public nombres: string;              // Nombres
  public apellidos: string;            // ApellIdos
  public fechaNacimiento: Date;        // Fecha_Nacimiento
  public fechaIngreso: Date;           // Fecha_Ingreso
  public idAFP: number;                // Id_AFP
  public afp: number;                   // AFP
  public idCargo: number;              // Id_Cargo
  public cargo: string;              // Cargo
  public sueldo: number;               // Sueldo
  public status: boolean;              // Status (1 = Activo, 0 = Inactivo)

  constructor(){}
}

File: api-net-core6.Domain/Entities/Empleado.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace api_net_core6.Domain.Entities
{
    public class Empleado
    {
        public int Id { get; set; }
        public string Nombres { get; set; }
        public string Apellidos { get; set; }
        public DateTime Fecha_Nacimiento { get; set; }
        public DateTime Fecha_Ingreso { get; set; }
        public int Id_AFP { get; set; }
        public string AFP { get; set; }
        public int Id_Cargo { get; set; }
        public string Cargo { get; set; }
        public decimal Sueldo { get; set; }
        public bool Status { get; set; } = true; // Activo por defecto
    }
}
